src/Symfony/Component/Security/Http/Logout/LogoutSuccessHandlerInterface.php

```diff
@@ -24,6 +24,8 @@
  * LogoutHandlerInterface instead.
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @deprecated since Symfony 5.1.
```

Comment:
can't we trigger a runtime notice for this one?
should we deprecate DefaultLogoutSuccessHandler?